framework/test/unit/modules/token/cc_method.spec.ts

```diff
@@ -409,7 +412,7 @@ describe('TokenInteroperableMethod', () => {
 			);
 		});
 
-		it('should deduct sending chain escrow account for fee and credit to receving chain escrow account if ccm failed', async () => {
+		it('should deduct sending chain escrow account for fee and credit to receving chain escrow account if ccm did not fail', async () => {
```

Comment:
For non failing CCM, receiving chain is to be credited, however the test verifies if the relayer account is verified; code logic suggests this scenario is when `ccmFailed: true` is the case.